elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -373,18 +375,27 @@ export class PeerPool extends EventEmitter {
 		return discoveredPeers;
 	}
 
-	public selectPeersAndConnect(
+	public triggerNewConnections(
 		peers: ReadonlyArray<P2PDiscoveredPeerInfo>,
```

Comment:
In the LIP, connection selection is based on each list with some condition, but would that be in another PR?

Reply:
We plan to implement that through this issue https://github.com/LiskHQ/lisk-sdk/issues/3346